### ISSUE TITLE
Demo Endpoint and Postman OpenAPI Import Returning "Error - No Records Found"

### ISSUE THREAD

**Issue by brendananders7** [GSA/auctions_api#26]
Hello,

I have used the demo link as well as imported the openAPI spec into Postman and used my personal key to attempt to retrieve data. Both of these resulted in the following response, despite there being many records found at https://autoauctions.gsa.gov/GSAAutoAuctions/VehicleListing

`{
    "Results": [
        {
            "status": "NOK",
            "error": "NO RECORDS FOUND"
        }
    ]
}`

**Comment by George-Gar** [comment]
i cant even sign up for it

**Comment by brendananders7** [comment]
@George-Gar I also had that issue, I signed up at this page to receive a key: https://open.gsa.gov/api/gsa-content/

**Comment by George-Gar** [comment]
@brendananders7 Thanks! You used this for the auctions api? I see that link is for a different api.

**Comment by brendananders7** [comment]
@George-Gar Yes I did use it and didn't get an authentication error, although it is useless if the API isn't working 😂

**Comment by 1ZoZ1** [comment]
How to get a images? "imageURL":"/api/v1/auction/image/41QSCI25564003.jpg"}
